sphinx_gallery/tests/test_full.py

```diff
@@ -26,11 +27,11 @@
 
 import pytest
 
-N_TOT = 13  # examples (plot_*.py in examples/**)
+N_TOT = 18  # examples (plot_*.py in examples/**)
 
 N_FAILING = 2
 N_GOOD = N_TOT - N_FAILING
-N_RST = 17 + N_TOT + 1  # includes module API pages, etc.
+N_RST = 17 + N_TOT + 1 + 12  # includes module API pages, etc.
 N_RST = '(%s|%s)' % (N_RST, N_RST - 1)  # AppVeyor weirdness
```

Comment:
I'll admit these numbers are completely empirical to me.  I don't understand what is included here and what should not be.  I added files to `tinybuild` and obviously they changed the totals, but I'm not clear where +12 comes from exactly, or how it relates to +5 in N_TOT.  Perhaps whoever architected these tests can tell me how to make this deterministic, and I can try.  (Note I also got screwed up for quite a while as I think you cannot run these tests standalone - you need to run the whole file?)

Reply:
It's likely not up to date anymore, and I agree it's not clear (even though I think I came up with it!). I think we probably want something like
```
N_EXAMPLES = ...  # examples/*.py files that get turned to RST
N_FAILING = 2
N_GOOD = N_EXAMPLES - N_FAILING
N_INDEX = ...  # index.rst files that SG generates
N_OTHER = ...  # other pages (API/modules, etc.)
N_RST = N_EXAMPLES + N_INDEX + N_OTHER
N_RST  = '(%s|%s)' % (N_RST, N_RST - 1)  # AppVeyor weirdness
```
or so, would that make sense to you? It might be a good idea to do this in `master` first to make sure it's working properly, though... I can do this sometime this week or next unless you want to try @jklymak

Reply:
Sure let me try as I looked at it all recently.  I tried doing what you have here but failed, but starting from master makes sense and then after that I can add one file at a time and get it to make sense.